### DISCUSSION TITLE
login Clear text transmission

### DISCUSSION THREAD

**Discussion by Luo-Xinmin** [cloudera/hue#3385] [Questions] [answered]
### Description

When logging in to the login page, it is transmitted in clear text. How to encrypt the transmitted account password?

http://IP:8889/hue/accounts/login?next=/

http://IP:8889/hue/accounts/login

csrfmiddlewaretoken: BNHRZnFovTgKNsA8BEOygfaQF3FfD9RxtsBbYv7jcZ6CfhMwfxF1pchF8i8vQzlW
username: admin
password: admin
next: /

**Comment by bjornalm** [comment, marked as answer]
Hi @Luo-Xinmin and thanks for reaching out. You should use SSL and connect to Hue via https. @Harshg999 please correct me if I'm missing something here.

**Reply by Luo-Xinmin** [reply]
Yes, I logged in through HTTP。
The account password requested for login is in clear text, which may cause interception and account password leakage
